### PR TITLE
Update introduction to Godot development debugging

### PR DESCRIPTION
**Pull request by mhilbrunner**
Updating parts to 3.0 and adding information on how to run tests & for Windows.

### CODE REVIEW

**Comment by NathanLovato** [comment]
Requesting you @akien-mga because I'm not sure if you want Windows-specific stuff to be in this document?
Also run/gdb commands don't exist on the Windows shells, even on git bash.

**Comment by mhilbrunner** [comment]
I can make changes if you have any suggestions. :)
Or feel free to close this if it's not wanted.

**Comment by NathanLovato** [comment]
What do you think @vnen ? Is this superfluous?

**Comment by vnen** [comment]
Yes, the Windows example is superfluous IMO.

File: development/cpp/introduction_to_godot_development.rst [hunk 1]
```diff
@@ -37,4 +37,29 @@ Or:
 .. code:: bash
 
     $ gdb godot
-    > run -e -path ~/myproject
+    > run -e ~/myproject/project.godot
+
+(With the path pointing to the project.godot or pointing to a specific scene to open the editor at.)
```
Comment (vnen) [inline development/cpp/introduction_to_godot_development.rst]: This is wrong, you cannot use the path to a scene directly. You can optionally put the scene as another argument, but not replacing the project path.

**Comment by mhilbrunner** [comment]
Closing then. :)

**Pull request closed without merging**